source/habitat_quality.rst

```diff
@@ -179,51 +179,54 @@ This section outlines the specific data used by the model. Please consult the In
+  
+  * *FUT_PATH*. The threat raster filepath for the future scenario where the filepath is relative to the threat CSV input. Entries are required if the future LULC was input, otherwise can be left empty if looking at current scenario only.
 
- Example: Hypothetical study with three threats. Agriculture (*Agric* in the table) degrades habitat over a larger distance than roads do, and has a greater overall magnitude of impact. Further, paved roads (*Paved_rd*) attract more traffic than dirt roads (*Dirt_rd*) and thus are more destructive to nearby habitat than dirt roads.
+ Example: Hypothetical study with three threats for both current and future scenarios. Agriculture (*Agric* in the table) degrades habitat over a larger distance than roads do, and has a greater overall magnitude of impact. Further, paved roads (*Paved_rd*) attract more traffic than dirt roads (*Dirt_rd*) and thus are more destructive to nearby habitat than dirt roads. Filepaths are relative to the Threat data table, so in this instance the current threats are found in the same directory as the table and the future threats are found in a sub directory adjacent to the Threat data table called *future*. Baseline threat filepaths are left blank because we do not have threat rasters for that scenario OR we have not included the baseline LULC in our model run altogether.
 
- ========   ======== ====== ===========
- THREAT     MAX_DIST WEIGHT DECAY
- ========   ======== ====== ===========
- Dirt_rd    2        0.1    linear
- Paved_rd   4        0.4    exponential
- Agric      8        1      linear
- ========   ======== ====== ===========
+ ========   ======== ====== =========== ============ =================  =======================
+ THREAT     MAX_DIST WEIGHT DECAY	BASE_PATH     CUR_PATH		FUT_PATH
+ ========   ======== ====== =========== ============ =================  =======================
+ Dirt_rd    2        0.1    linear		      dirt_rd.tif       future/dirt_rd_fut.tif
+ Paved_rd   4        0.4    exponential		      paved_rd.tif	future/paved_rd_fut.tif
+ Agric      8        1      linear		      agric_rd.tif	future/agric_rd_fut.tif
+ ========   ======== ====== =========== ============ =================  =======================
 
 |
 
+* **Threat Rasters Information**: GIS raster files of the distribution and intensity of each individual threat, with values between 0 and 1. You will have as many of these maps as you have threats and the raster filepath should be defined in the **Threats data** table. These threat maps should cover the area of interest of the study, such that the area of interest is inset from the edges of the threat raster by the width of the maximum threat distance. Otherwise, locations near the edge of the area of interest may have inflated habitat quality scores, because threats outside the area of interested are not properly accounted for.
```

Comment:
See my note above about possibly keeping this "raster coverage" discussion in only one place and linking to it as needed.